### PR TITLE
feat(ecl-editor): init .ecl-editor -INNO-642

### PR DESCRIPTION
**Pull request by degliwe**
start moving heading rules to mixins and reuse them for .ecl-editor element

Output should be:
- .ecl-editor namespace plus style for defautl element in europa.css
- new css style sheet editor-europa

### CODE REVIEW

File: framework/components/ecl-blockquotes/_blockquotes.scss [hunk 1]
```diff
@@ -3,19 +3,10 @@
...
-  line-height: 1.5;
-  margin-left: 0;
-  margin-right: 0;
-  max-width: 24em;
-  overflow: hidden;
-  padding: 0.6em 0 0.6em 1.2em;
-  position: relative;
+  @include blockquotes();
 
   // Drupal fix.
```
Comment (yhuard) [inline framework/components/ecl-blockquotes/_blockquotes.scss]: Do we really have a `Drupal fix`? o_O

File: framework/components/ecl-blockquotes/_blockquotes-mixins.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@mixin blockquotes() {
+  border-left: 0.417em solid $ecl-color-secondary;
```
Comment (yhuard) [inline framework/components/ecl-blockquotes/_blockquotes-mixins.scss]: `0.417em`... maybe it's time to refactor this. I think it's okay to give it an absolute value in pixels (still better than `0.417em`)

File: framework/components/ecl-blockquotes/_editor-blockquotes.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import 'blockquotes-mixins';
+
+.ecl-editor blockquote {
+  @include blockquotes();
+
+  // Drupal fix.
```
Comment (yhuard) [inline framework/components/ecl-blockquotes/_editor-blockquotes.scss]: I'm getting concerned by those `Drupal fix`, didn't know we still had them 😟

File: framework/components/ecl-links/_links-mixins.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+@mixin link() {
+  color: $ecl-color-primary;
+  line-height: 1.66;
+  text-decoration: underline;
+
+  &:hover,
+  &:focus {
+    color: $ecl-color-shade;
```
Comment (yhuard) [inline framework/components/ecl-links/_links-mixins.scss]: We could take this approach one step further and decouple the `&:` rules too

File: framework/components/ecl-tables/_tables.scss [hunk 5]
```diff
@@ -84,32 +39,6 @@
 
 @media (min-width: map-get($ecl-grid-breakpoints, 'md') + 1) {
```
Comment (yhuard) [inline framework/components/ecl-tables/_tables.scss]: `@media (min-width: map-get($ecl-grid-breakpoints, 'md') + 1)` should be replaced by `@include ecl-media-breakpoint-up(md)`

File: ecl-builder.config.js [hunk 6]
```diff
@@ -23,6 +23,13 @@ module.exports = {
         sourceMap: isProd ? 'file' : true,
       },
     },
+    {
+      entry: path.resolve(__dirname, 'framework/editor-index.scss'),
+      dest: path.resolve(__dirname, outputFolder, 'styles/editor-europa.css'),
+      options: {
+        sourceMap: isProd ? 'file' : true,
```
Comment (yhuard) [inline ecl-builder.config.js]: Now `normalize` is missing...

**Comment by yhuard** [comment]
Maybe we should take the chance to add tests for those content-specific rules ;)

File: framework/components/ecl-blockquotes/_blockquotes.scss (hunk 1 above)
Comment (degliwe) [inline framework/components/ecl-blockquotes/_blockquotes.scss]: it seems that we don't have drupal fix, but the comment was misleading.

File: framework/components/ecl-blockquotes/_editor-blockquotes.scss (hunk 3 above)
Comment (degliwe) [inline framework/components/ecl-blockquotes/_editor-blockquotes.scss]: it seems that we don't have drupal fix, but the comment was misleading.

**Pull request merged**